+++ ./src/lib/googletest/googletest/include/gtest/internal/gtest-port-arch.h	2020-02-08 21:53:36.258302493 +0100
@@ -71,6 +71,8 @@
 # if TARGET_OS_IPHONE
 #  define GTEST_OS_IOS 1
 # endif
+#elif defined __GENODE__
+# define GTEST_OS_GENODE 1
 #elif defined __DragonFly__
 # define GTEST_OS_DRAGONFLY 1
 #elif defined __FreeBSD__
